src/Screens/HomeScreen/HomeScreen.js

```diff
@@ -0,0 +1,69 @@
+import './HomeScreen.css'
+
+import { Header } from '../../Components/Header/Header'
+import { Footer } from '../../Components/Footer/Footer'
+import { IcBaselinePlayCircleOutline } from '../../assets/logos'
+
+import { useCategory } from '../../ActionProviders/categoryActions'
+import { useEffect } from 'react'
+
+import { Loader } from '../../Components/Loader/Loader'
+import { Error } from '../../Components/Error/Error'
+
+import '../../assets/logo.css'
+
+const HomeScreen = () => {
+	const {
+		categories,
+		categoriesLoading,
+		categoriesError,
+		fetchCategories,
+	} = useCategory()
+
+	console.log(categories, categoriesLoading, categoriesError)
```

Comment:
suggestion: remove `console.log`

Reply:
Thankyou will note it from next time to remove console logs while doing PRs